src/table/table-row.jsx

```diff
@@ -94,8 +94,9 @@ const TableRow = React.createClass({
 
     let styles = {
       root: {
-        borderBottom: '1px solid ' + this.getTheme().borderColor,
-        color: this.getTheme().textColor,
+        borderBottom: '1px solid ' + theme.borderColor,
+        color: theme.textColor,
+        height: theme.height
```

Comment:
@oliviertassinari But [tableRowColumn has a default height](https://github.com/zachguo/material-ui/blob/master/src/styles/getMuiTheme.js#L178) too.
I agree we should have adaptive cell height & width, however I think we should do it in another PR.